### ISSUE TITLE
Can not properly theme send button

### ISSUE THREAD

**Issue by simonsickle** [apptentive/apptentive-android#199]
The send button can not be properly themed to whatever we want. https://github.com/apptentive/apptentive-android/blob/master/apptentive/src/main/res/layout/apptentive_survey.xml#L65

Only typeFace and TextSize are used here. https://github.com/apptentive/apptentive-android/blob/master/apptentive/src/main/res/values/styles.xml#L156

Text color can not be set as a part of this theme, so in my case in night mode the button text goes black when I need it to stay as a white color in day or night theme.

**Comment by CaseyApptentive** [comment]
Hi @sicklepilot! I apologize for the issue with theme. Happy to point you in the right direction. Are you using Material Design or AppCompat?

We've recently updated our docs for help with the two most common types of issues with Android. Apologies if you've seen this already, but I wanted to share just in case: https://learn.apptentive.com/knowledge-base/android-interface-customization/#troubleshooting

**Comment by sicklepilot** [comment]
Hi, we’re using Material Components for our base theme now (migration from AppCompat to support addition of FABs)

The issue here is with this one send button, as the text color is not able to be overridden. If we got another element added to that theme (android:textColor) and another attribute like the other fields, we could easily select the text color we want via overrides. 

I cloned the library to confirm this and could raise a PR if it would be accepted back to mainline. Let me know!
